config/routes.rb

```diff
@@ -106,6 +106,9 @@
     end
     collection { get :search }
   end
+  resources :drafts, only: [:create, :destroy] do
+    collection { post :preview }
```

Comment:
Do we have a `drafts#preview`? I don't think this is used based on the controller!

Reply:
Oh yeah that's leftover from when I was also trying to get a preview path working, i should probably remove it. (That will have another PR later probably but didn't need to be in this one xD)

Reply:
Ooh ok! Yeah, I think we can remove it from this PR but otherwise LGTM!